src/scss/helpers.module.scss

```diff
@@ -0,0 +1,60 @@
+@import './breakpoints.module.scss';
+
+// Base font size. If need be, can be made responsive.
+$base-font-size: 16;
+
+// Spacer keys are the PX values used in the Figma files:
+// 4, 8, 16, 24, 32, 64, 80, 120
+// prettier-ignore
+$spacers: (
```

Comment:
@kevee should I add a px fallback? To me it seems overkill regarding https://caniuse.com/#search=rem but it's a quick fix to do

Reply:
I think rem is fine.